client/src/components/LobbyChat.svelte

```diff
@@ -0,0 +1,70 @@
+<script lang="ts">
+  import Modal from "./Modal.svelte";
+  import { onMount, onDestroy } from "svelte";
+  import { authStore } from "$stores/auth";
+  import { onSnapshot, orderBy, query, QueryDocumentSnapshot } from "firebase/firestore";
+  import type { User } from "firebase/auth";
```

Comment:
these import are not used

you should do `npm run lint` in the cliend directory and try to fix the warnings if they are related to code you wrote